src/components/common/ColonyDecisions/DecisionPreview/DecisionData/DecisionData.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { Decision } from '~types';
+import { AppContextValues } from '~context/AppContext';
+
+import { DecisionContent, DecisionNotFound } from '../DecisionData';
+
+const displayName = 'common.ColonyDecisions.DecisionPreview.DecisionData';
+
+export interface DecisionDataProps {
+  decision?: Decision;
+  user: AppContextValues['user'];
```

Comment:
The point here is to create the dependency, because it's being passed the user object that comes from app context. Say the type of the user were to change in app context, this would save me having to go and update that type everywhere I pass the user object from app context as a prop.

Reply:
Should this component know about `AppContext` at all? I think it should be happy just accepting `user` of type `User`, no matter where it comes from. I'd argue this makes refactoring more difficult.
But more importantly, this isn't a common pattern for the CDapp - for example, we don't use `ColonyContextValues['colony']` instead of `Colony` from `~types`.

Reply:
I agree with your second point, will change.